CHANGELOG.md

```diff
@@ -8,6 +8,33 @@ Read our [guidelines to categorize a change](https://biomejs.dev/internals/versi
 New entries must be placed in a section entitled `Unreleased`.
 Read our [guidelines for writing a good changelog entry](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#changelog).
 
+## Unreleased
+
+### Analyzer
+
+### CLI
+
+### Configuration
+
+### Editors
+
+### Formatter
+
+#### Bug fixes
+
+- Fix [#1039](https://github.com/biomejs/biome/issues/1039). Check unicode width instead of number of bytes when checking if regex expression is a simple argument. Contributed by @kalleep
+
+This no longer breaks.
+```js
+s(/🚀🚀/).s().s();
+```
+
```

Comment:
This should be indented to attach the content to the bullet point.
```suggestion

  This no longer breaks.

  ```js
  s(/🚀🚀/).s().s();
  ```

```